frontend/public/components/cluster-service-class.tsx

```diff
@@ -128,14 +89,9 @@ export type ClusterServiceClassPageProps = {
 };
 
 export type ClusterServiceClassDetailsProps = {
-  obj: K8sResourceKind
+  obj: K8sResourceKind,
```

Comment:
Are we supposed to add trailing commas?  I ask because I removed them once, but ended up putting them back

Reply:
Prefer adding the trailing comma when items are split across lines. It's perfectly valid in ES6 and makes it easier to add items / reorder properties.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
they are allowed based on https://github.com/Microsoft/TypeScript/pull/20599, so I guess its up to us 
+1 for having them